src/components/homePage/homePage.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import { Link } from "react-router-dom";
```

Comment:
We can use Link provided by material-ui too instead of using the react-router-dom Link if we are using MaterialUI for uniformity.
https://material-ui.com/components/links/